.factory/automation.yml

```diff
@@ -41,6 +41,7 @@ build:
         bazel build //...
         bazel run @vaticle_dependencies//tool/checkstyle:test-coverage
         bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=errors
+        bazel test $(bazel query 'kind(rustfmt_test, //...)') --@rules_rust//:rustfmt.toml=//:rustfmt_config
```

Comment:
The only way to provide our custom `rustfmt.toml` to the `rustfmt_test` rules that I found is through the aspect.

Reply:
The way I understand it, this invocation sets the `rustfmt.toml` `label_flag` provided for `rustfmt_aspect` that is used by `rustfmt_test`.

https://bazel.build/extending/config#label-typed-build-settings
https://github.com/bazelbuild/examples/tree/HEAD/configurations/label_typed_build_setting